### PR TITLE
[ISSUE-3813][Feature] Introduce member management e2e test

### LINKED ISSUE #3813: [Feature] Add Member Management e2e test cases
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Add Member Management e2e test cases:

- createMember
- deleteMember
- editMember

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhengkezhou1**
## What changes were proposed in this pull request

Issue Number: close #3813 

add basic member management test cases.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

(or)

This change is already covered by existing tests, such as (please describe tests).

(or)

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (no)

### CODE REVIEW

**Comment by zhengkezhou1** [comment]
@SbloodyS PLTA.

**Comment by wolfboys** [comment]
Could you please resolve the conflict first?

File: streampark-e2e/streampark-e2e-case/src/test/resources/docker/basic/docker-compose.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ version: "3.8"
 
 services:
   streampark:
-    image: apache/streampark:ci-basic
+    image: apache/streampark:dev
```
Comment (SbloodyS) [inline streampark-e2e/streampark-e2e-case/src/test/resources/docker/basic/docker-compose.yaml]: Please avoid unnessnary change.

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**